spicy/toolchain/src/compiler/codegen/grammar-builder.cc

```diff
@@ -130,7 +130,7 @@ struct Visitor : public visitor::PreOrder {
             if ( const auto& ctor = n->ctor() ) {
                 auto prod = productionForCtor(ctor, n->id());
                 auto m = prod->meta();
-                m.setField(pf->currentField(), true);
+                m.setField(pf->currentField(), false);
```

Comment:
I'm not sure we actually *require* this change (I didn't see difference in effect either), but conceptually it seems right. My fixup makes the Skip production the top-level production that's in charge of the field. With that, the ctor doesn't need (should) have the flag set anymore, because it's wrapped inside the Skip. That's also why I don't think we need to query the meta data in the ctor for this.